package.json

```diff
@@ -0,0 +1,606 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "Todolist",
+  "devDependencies": {
+    "html-webpack-plugin": "^5.5.0",
+    "webpack": "^5.61.0",
+    "webpack-cli": "^4.9.1"
```

Comment:
- [ ] `webpack-dev-server` should exist on your project dependencies, kindly add it 😄, otherwise your `npm run start` command will not work properly.

Reply:
This issue is well resolved! ✔️